libs/src/sdk/api/developer-apps/types.ts

```diff
@@ -0,0 +1,12 @@
+export type CreateDeveloperAppRequest = {
+  userId: string
+  name: string
+  // description: string
+  /** True if the user only needs an API Key + Secret for personal use on their own account only */
```

Comment:
suuper nit but for third parties it might be nice to read this as "Set this to true if you're only using the API key and secret for your own user account" or something with a more instructive voice rather than descriptive